app/views/frontend/issue_index.haml

```diff
@@ -1,4 +1,4 @@
-- Issue.reorder(:volume).last.volume.downto(1) do |i|
+- Issue.published.reorder(:volume).last.volume.downto(1) do |i|
```

Comment:
These changes seem to be unrelated to this pull request?

Reply:
Oops, these were from something else that I was trying. Fixed now.